### DISCUSSION TITLE
제페토 월드 내에서 웹페이지를 표기하고 싶습니다.

### DISCUSSION THREAD

**Discussion by devinhyeok** [naverz/zepeto-studio-kor#267] [5. Q&A]
![image](https://user-images.githubusercontent.com/94621970/150982703-2e8fc628-f7b6-48ad-930f-277bdca35213.png)
![image](https://user-images.githubusercontent.com/94621970/150982717-4e07824d-7ced-4a74-a9e8-8e8494c42ce3.png)

삼성갤럭시맵에 들어가면 위에 사진처럼 상호작용하여 인게임에서 웹사이트에 들어갈 수 있던데 공식문서에서 방법을 찾을수 없습니다.
어떻게 하나요? 
삼성갤럭시맵에만 별도의 작업을 해준것인가요?
아니면 제가 모르는 마법같은 함수가 있는건가요?

**Comment by x4cf** [comment]
외부링크는 사용 불가능합니다.
가이드 월드규정 2.5.5를 참고해주세요

**Reply by devinhyeok** [reply]
가이드 월드규정 2.5.5가 어디에 있나요? 링크좀 주실수 있나요?

**Reply by DoLongJu** [reply]
https://studio.zepeto.me/guides/world-guidelines
가이드 여깄습니다!

**Reply by devinhyeok** [reply]
감사합니다!
